source/components/LocalNavigation/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import FandomContentWell from '../FandomContentWell';
+
+import './styles.scss';
+
+export const ACTIVE_CLASS_NAME = 'is-active';
+
+/**
+ * LocalNavigation is a simple navigation than can be used with links or `react-router`.
+ *
+ * To mark child as active use exported class `ACTIVE_CLASS_NAME`.
+ */
+class LocalNavigation extends React.PureComponent {
+    modifyChildren = (child) => {
```

Comment:
minor: this seems to only operate on a single child, so a better name might be `modifyChild`

Reply:
fixed, thanks!